### PR TITLE
[Test lib] Added test case for verifying if pachyderm is installed

### PR DESCRIPTION
**Pull request by risusing**

### CODE REVIEW

**Comment by bdattoma** [comment]
lgtm. Please run robotidy/robocop

File: tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+*** Test Cases ***
+Verify if it is possible to install Pachyderm
+    [Tags]  ODS-1137
+    Navigate to OperatorHub
+    Get The Number of Operator Available    pachyderm
+    Install Operator    pachyderm
+    Navigate to Installed Operators
+    Search Installed Operator   pachyderm
+    Is Operator Installed   pachyderm
```
Comment (tarukumar) [inline tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot:16]: Line number 11-16 could be replaced with **_Check And Install Operator in Openshift_**

Comment (bdattoma) [reply tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot:16]: Tier2 @tarukumar

File: tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Resource        ../../../Resources/OCP.resource
+Test Setup      Dashboard Test Setup
+Test Teardown   Dashboard Test Teardown
+
+*** Variables ***
+${pachyderm_container_name}     Pachyderm
+${pachyderm_appname}            pachyderm
+
+*** Test Cases ***
+Verify Inetl AIKIT Operator Can Be Installed Using OpenShift Console
```
Comment (tarukumar) [inline tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot:13]: Test case name should be changed to Pychyderm. Here it is mentioned intel aikit

File: tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    [Tags]  Tier-2
```
Comment (tarukumar) [inline tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot:14]: Tier tags shouldn't have "-" character. IIt should be **Tier2** without space. Please refer style guide for reference https://docs.google.com/document/d/11ZJOPI1uq-0Wl6a2V8fkAv_TQhfzp9t_IjXAheaJxmQ/edit#heading=h.4a2fb94fltk2

File: tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot (hunk 2 above)
Comment (risusing) [reply tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot:13]: sorry actually I was copying it and it happened by mistakenly. I am changing it.

**Comment by risusing** [comment]
https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/460/
Jenkins job run link.

**Review by bdattoma — APPROVED** [review]

File: tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+
+*** Variables ***
+${pachyderm_container_name}     Pachyderm
+${pachyderm_appname}            pachyderm
+
+
+*** Test Cases ***
+Verify Pachyderm Can Be Installed Using OpenShift Console
+    [Tags]    Tier2    ODS-1137
```
Comment (lugi0) [inline tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot:17]: Could you use the usual formatting for Tags?

```
[Tags]    Tier2
...         ODS-XYZ
```

lgtm other than this

Comment (jgarciao) [reply tests/Tests/600__ai_apps/670__pachyderm/670__pachyderm.robot:17]: To fix in another PR

**Review by lugi0 — APPROVED** [review]

**Pull request merged**